Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -113,8 +113,12 @@ public struct PopupView: View {
                         )
                     }
                 } else {
-                    VStack(alignment: .center) {
-                        Text("Evaluating popup expressions…", bundle: .toolkitModule)
+                    HStack(alignment: .center, spacing: 10) {
+                        Text(
+                            "Evaluating popup expressions",
+                            bundle: .toolkitModule,
+                            comment: "A label indicating popup expressions are being evaluated."
+                        )
                         ProgressView()
                     }
```

Comment:
The translators reported garbled characters in their translation software on this string. We can try alternatives like using the unicode value for the horizontal ellipsis instead but with the `ProgressView()` in an `HStack` right next to this I think we're okay without the `…` all together.

<img width="377" alt="Screenshot 2023-08-30 at 16 54 14" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/187e59ce-664d-4ff1-a3ee-bfba4c2246c0">